doc/Developing/os/Health-Information.md

```diff
@@ -108,6 +108,9 @@ The only sensor we have defined here is airflow. The available options are as fo
 - `index` (optional): This is the index value we use to uniquely
   identify this sensor. `{{ $index }}` will be replaced by the `index`
   from the snmp walk.
+  For the 'count' sensor class, it is possible to prefix the index with
```

Comment:
Why are you putting this setting in the index?  That doesn't seem like the right place for it.

Reply:
The other option I see is to create a new column in the sensor DB. It would definitly be a little cleaner. Any other idea ?

Reply:
@murrant Should I go for the variant where I add a column (enum) with the type to the sensor table ? 
If yes, I would also try to add the unit to the sensor, so we could (at least for "count") override the unit as well.

Reply:
It doesn't need to be stored in the database at all, it can just be in the yaml.

Reply:
That would make it for discovery, but I also need this information during polling (so I can do the diff between previous and current value). And the YAML file is not read again during each polling session (which is good for perf issues). 
So unless the information is stored in the DB, I don't see a way.

Reply:
I think it should be declared separately at least in the yaml.